sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -1048,4 +1154,40 @@ private void completeDeferredMessages(ServiceBusReceiverAsyncClient client, Serv
         }
 
     }
+
+    protected ServiceBusMessage getMessage(String messageId, boolean isSessionEnabled, AmqpAnnotatedMessage amqpPropertiesToSet) {
```

Comment:
We don't need a protected method for this. It is only used once. Just put it in the arrangement.